airbyte-webapp/src/components/connectorBuilder/StreamTestingPanel/ConfigMenu.tsx

```diff
@@ -62,7 +62,7 @@ export const ConfigMenu: React.FC<ConfigMenuProps> = ({ className, testInputJson
               onClick={() => setIsOpen(true)}
               disabled={
                 !jsonManifest.spec ||
-                Object.keys(jsonManifest.spec.connection_specification.properties || {}).length === 0
+                Object.keys(jsonManifest.spec.connection_specification?.properties || {}).length === 0
```

Comment:
Small bugfix - the page would crash if the `spec` did not contain `connection_specification`, and this change prevents that crash from occurring